main/pipeline/pipeline.cc

```diff
@@ -295,11 +296,17 @@ indexOneWithPlugins(const options::Options &opts, core::GlobalState &gs, core::F
 vector<ast::ParsedFile> incrementalResolve(core::GlobalState &gs, vector<ast::ParsedFile> what,
                                            const options::Options &opts) {
     try {
+#ifndef SORBET_REALMAIN_MIN
+        if (opts.stripePackages) {
+            Timer timeit(gs.tracer(), "incremental_packager");
+            what = packager::Packager::runIncremental(gs, move(what));
+        }
+#endif
         {
             Timer timeit(gs.tracer(), "incremental_naming");
-            auto emptyWorkers = WorkerPool::create(0, gs.tracer());
             core::UnfreezeSymbolTable symbolTable(gs);
             core::UnfreezeNameTable nameTable(gs);
+            auto emptyWorkers = WorkerPool::create(0, gs.tracer());
```

Comment:
Is this variable unused?

Reply:
Ah nvm i didn't see that it showed up in previous side of the diff, and it's used a line below.

Reply:
It's used on the next line. I moved it closer to where it is used.